tests/Api/Admin/OrdersTest.php

```diff
@@ -79,17 +114,109 @@ public function it_gets_orders_for_customer(): void
         /** @var CustomerInterface $customer */
         $customer = $fixtures['customer_tony'];
 
-        $this->client->request(
-            method: 'GET',
-            uri: '/api/v2/admin/orders?customer.id=' . $customer->getId(),
-            server: $this->buildHeaders('api@example.com'),
+        $this->requestGet(
+            uri: '/api/v2/admin/orders',
+            queryParameters: ['customer.id' => $customer->getId()],
         );
 
-        $this->assertResponse(
-            $this->client->getResponse(),
-            'admin/order/gets_orders_for_customer_response',
-            Response::HTTP_OK,
+        $this->assertResponseSuccessful('admin/order/gets_orders_for_customer_response');
+    }
+
+    /**
+     * @test
+     *
+     * @dataProvider provideDates
+     *
+     * @param string[] $checkoutsCompletedAt
+     * @param array{filterType: FilterTypes, date: string} $requestedLimit
+     *
+     * @throws \Exception
+     */
```

Comment:
This is a test. Are these comments necessary?

Reply:
IDE complains, but perhaps I should configure it to exclude test namespaces.